src/esp/gfx/LightSetup.cpp

```diff
@@ -76,12 +76,34 @@ LightSetup getLightsAtBoxCorners(const Magnum::Range3D& box,
                     {{box.backBottomLeft(), w}, lightColor},
                     {{box.backBottomRight(), w}, lightColor}};
 }
+
 LightSetup getDefaultLights() {
-  return LightSetup{{{0.0, -0.5, -1.0, 0.0}, {2.4, 2.4, 2.4}},    // forward
-                    {{0.0, -0.5, 1.0, 0.0}, {2.4, 2.4, 2.4}},     // backward
-                    {{1.0, -1.0, 0.0, 0.0}, {2.0, 2.0, 2.0}},     // top left
-                    {{-1.0, -1.0, -1.0, 0.0}, {2.0, 2.0, 2.0}},   // top right
-                    {{0.0, 1.0, 0.0, 0.0}, {0.64, 0.64, 0.64}}};  // bottom
+  return LightSetup{
+      {{0.0, -0.1, -1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // -z
+      {{0.0, -0.1, 1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // +z
+      {{-1.0, -0.1, 0.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // -x
+      {{+1.0, -0.1, -1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // +x
+      {{0.0, -1.0, 0.0, 0.0},
+       {0.6, 0.6, 0.6},
+       LightPositionModel::Global},  // downward
+
+      // camera-relative
+      {{0.0, -0.1, -1.0, 0.0},
+       {0.3, 0.3, 0.3},
+       LightPositionModel::Camera},  // forward
+      {{0.0, -0.3, 1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Camera},  // backward (for glancing speculars)
+
+  };
```

Comment:
Just so everyone is aware -- this is 7 lights instead of 5, which makes the fragment shader roughly as much as 40% more computationally expensive. 

Personally I don't think that many lights is needed, even a three-point lighting was often enough to get a balanced output.

Reply:
I was aware of the potential impact of these two additional camera-related lights, but their intention is specifically to give easily visualized specular/textural response for those materials that have texture-based surface normals, clearcoat texture and cc normal texture and anisotropy (so far).  

If folks think this is too expensive, I can remove the two camera facing lights, and instead provide a lighting config that would define the lights in this manner that folks making demos could consume.

Reply:
It is also pretty easy to just author a new light config with fewer lights if the default is considered too computationally expensive.

Reply:
I think that one directional light and an ambient light would suffice as a default.

Reply:
A single directional light will not look good at all for PBR scenes (I've tried it), and ambient light is neither used nor useful in PBR, since it lacks a light vector.  Of course it probably will be fine with Phong lighting, so perhaps we should set up a different default for Phong? 

At the -very least-, I would suggest the 5 base global directional lights I have in the current default setup for PBR.